redfish-core/lib/log_services.hpp

```diff
@@ -6275,14 +6275,23 @@ inline void getFullAuditLogAttachment(
         return;
     }
 
-    long long int size = lseek(fd, 0, SEEK_END);
+    off_t size = lseek(fd, 0, SEEK_END);
     if (size == -1)
```

Comment:
This new logic seems appropriate and correct - https://pubs.opengroup.org/onlinepubs/9699919799.2016edition/functions/lseek.html